lib/omnitrade_client.dart

```diff
@@ -0,0 +1,24 @@
+library omnitrade_client;
+
+import 'package:chopper/chopper.dart';
+import 'src/services.dart';
```

Comment:
What do you think about leaving package dependencies imports at the top and local dependencies in the lines below it?